RecoBTag/Combined/python/DeepCSVJetTags_cfi.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+DeepCSVJetTags = cms.EDProducer(
```

Comment:
module instances should start with a lower case letter.
Please change and rename the file as well.